go.mod

```diff
@@ -26,11 +26,14 @@ require (
 	go.mongodb.org/mongo-driver v1.11.6
 	go.uber.org/zap v1.24.0
 	golang.org/x/crypto v0.11.0 // indirect
-	golang.org/x/sys v0.10.0
+	golang.org/x/sys v0.12.0
 	google.golang.org/protobuf v1.30.0 // indirect
 )
 
```

Comment:
any reason for this to be re-introduced?

Reply:
No I will return it to it's default